airbyte-integrations/connectors/source-webflow/source_webflow/spec.yaml

```diff
@@ -0,0 +1,29 @@
+documentationUrl: https://docsurl.com/TBD
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Webflow Spec
+  type: object
+  required:
+    - api_key
+    - site_id
+    - accept_version
+  additionalProperties: false
+  properties:
+    site_id:
+      title: Site id
+      type: string
+      description: "The id of the Webflow site you are requesting data from. See https://developers.webflow.com/#sites"
+      order: 0
+    api_key:
+      title: API token
+      type: string
+      description: "The API token for authenticating to Webflow. See https://university.webflow.com/lesson/intro-to-the-webflow-api"
+      example: "Bearer <a long hex sequence>"
+      order: 1
+    accept_version:
```

Comment:
Apologies, this comment is referring to the accept_version field, not the api_key field.

Reply:
Correct, the user doesn't need to pass in the field, which is why it is hidden. 

Specifying this as a constant in  spec.yaml provides a "easier" mechanism to update the version of the Webflow API to be used, by modifying it directly in the spec file,  as opposed to hard-coding it in the code. 

This could equally be defined as a CONST value in the code if the current approach is not preferred.

Reply:
I've moved this definition into a constant in the code.